Slam/Queue/APIManagedQueue.swift

```diff
@@ -9,10 +9,23 @@ public class APIManagedQueue: QueueManager {
         self.queueView = queueView
     }
 
+    public init() {
+        self.api = SlamAPI(httpClient: AsynchronousHTTPClient())
+        self.queueView = nil
+    }
+
     public func loadQueue(onSuccess: () -> Void) {
         api.getQueue({(matches: [Match]) in
-            self.queueView.showMatches(matches)
+            self.queueView!.showMatches(matches)
             onSuccess()
         })
     }
+
+    public func removeMatch(matchID: String) {
+        api.removeMatchFromQueue(matchID, onSuccess: { () -> Void in self.queueView!.removedMatch()})
```

Comment:
This should be split across multiple lines. It's kind of hard to read as-is. After playing around with the code, it also seems the function signature is not needed.

Like this:
```swift
api.removeMatchFromQueue(matchID, onSuccess: {
  self.queueView!.removedMatch()
})
```